.sops.yaml

```diff
@@ -0,0 +1,11 @@
+keys:
```

Comment:
Brody @faultables give me you prefer key here

Reply:
https://github.com/faultables.keys would good

Reply:
@faultables gpg aja brody

Reply:
coba ini https://github.com/faultables.gpg